eox_core/api/v1/serializers.py

```diff
@@ -6,7 +6,9 @@
 
 from collections import OrderedDict
 
+import six
```

Comment:
We dropped support for Python 2 in version 4.0.0

Reply:
@MoisesGSalas  I fixed it, thanks